src/app.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+const fs = require('fs');
+
+function copyFiles() {
+  const [fileSource, fileDestination] = process.argv.slice(2);
```

Comment:
Consider adding a check to ensure that `fileSource` is provided. Currently, the code only checks if `fileDestination` is provided.